main_test.go

```diff
@@ -2,11 +2,15 @@ package main
 
 import (
 	"os"
+	"reflect"
 	"slices"
 	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
 )
 
-func Test_getVars(t *testing.T) {
+func TestGetVars(t *testing.T) {
```

Comment:
Were we following a different convention and now going more conventional go?

Reply:
Thanks for asking. This is one area that doesn't seem to have agreement in the Go community. Goland generates test functions using an underscore in at least 2 situations: lowercase function names and struct-attached functions. I made this edit because we have a SonarQube rule that prohibits underscores in function names. Rules can't vary by test vs non-test code, so I just decided to remove the underscores. Other options would be to individually mark them as acceptable or to disable scanning of all test files.